modules/porous_flow/src/fvbcs/FVPorousFlow1PhaseAdvectiveFluxBC.C

```diff
@@ -0,0 +1,83 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "FVPorousFlow1PhaseAdvectiveFluxBC.h"
+#include "PorousFlowDictator.h"
+#include "FVDirichletBC.h"
+
+registerADMooseObject("PorousFlowApp", FVPorousFlow1PhaseAdvectiveFluxBC);
+
+InputParameters
+FVPorousFlow1PhaseAdvectiveFluxBC::validParams()
+{
+  InputParameters params = FVFluxBC::validParams();
+  RealVectorValue g(0, 0, -9.81);
+  params.addParam<RealVectorValue>("gravity", g, "Gravity vector. Defaults to (0, 0, -9.81)");
```

Comment:
how often do you do 2D @cpgr ?

this default isnt great for 2D, I d almost want to error if this is used in 2D

Reply:
Yeah, I just used the same default we have in the other porous flow objects that exist. 

I guess we could either not have a default, or do something fancier like check the mesh dimension and then set an appropriate value?

Reply:
both are an option. It s not tied to this object then, can be figured out later